kornia/augmentation/_2d/intensity/random_channel_dropout.py

```diff
@@ -0,0 +1,95 @@
+from random import randint, sample
+from typing import Any, Dict, Optional, Tuple, Union
+
+import numpy as np
+from torch import Tensor, clone
+
+from kornia.augmentation._2d.intensity.base import IntensityAugmentationBase2D
+from kornia.core.check import KORNIA_CHECK
+
+
+class RandomChannelDropout(IntensityAugmentationBase2D):
+    """Randomly Drop Channels in the input Image.
+
+    Args:
+        channel_drop_range (int, int): range from which we choose the number of channels to drop.
```

Comment:
```suggestion
        channel_drop_range: range from which we choose the number of channels to drop.
```
types are auto filled in during the docs generation